src/Migrations/Version20250520170950.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20250520170950 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Remove plan to claim GA for one charity';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql(<<<'SQL'
+            UPDATE Donation 
+            SET matchbot.Donation.tbgShouldProcessGiftAid = 0
+            WHERE Donation.campaign_id = 8963 and Donation.donationStatus in ('paid', 'collected')
```

Comment:
```suggestion
            WHERE Donation.campaign_id = 8963 and Donation.donationStatus in ('Paid', 'Collected')
```
Marginally clearer for future code searchers? But collation is `_ci`, not a blocker.